admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -2,35 +2,43 @@
-with.
-
-.. note:: Encrypting files increases their size by roughly 35%, so you must take 
-   this into account when you are provisioning storage and setting storage 
-   quotas. User's quotas are based on the unencrypted file size, and not the 
-   encrypted file size.
+In ownCloud 8.1 the Encryption app has been changed to the ownCloud Default 
+Encryption Module. It has a number of changes and improvements, including:
+
+* When encryption is enabled, all files are no longer encrypted at user's first 
+  logins because this caused timeouts on large installations. Instead, only 
+  files that are created or updated after encryption has been enabled are 
+  encrypted.
+
+* The  "decrypt all" option in the Personal settings has been removed, also for 
+  performance reasons.
+
+* A new option for users to enable/disable encryption on a per mount-point 
+  basis.
+
+The Default Encryption Module encrypts files stored on the ownCloud server, and 
+files on remote storage that is connected to your ownCloud server. Encryption 
+and decryption are performed on the ownCloud server. All files sent to remote 
+storage (for example Dropbox and Google Drive) will be encrypted by the ownCloud 
+server, and upon retrieval, decrypted before serving them to you and anyone you 
+have shared them with.
+
+.. note:: Encrypting files increases their size by roughly 35%, so you must 
+   take this into account when you are provisioning storage and setting 
+   storage quotas. User's quotas are based on the unencrypted file size, and 
+   not the encrypted file size.
 
 When files on external storage are encrypted in ownCloud, you cannot share them 
 directly from the external storage services, but only through ownCloud sharing 
 because the key to decrypt the data never leaves the ownCloud server.
 
-The main purpose of the Encryption app is to protect users' files on remote 
-storage, and to do it easily and seamlessly from within ownCloud. 
-
-The Encryption app generates a strong encryption key, which is unlocked by 
-user's passwords. So your users don't need to track an extra password, but 
-simply log in as they normally do.
-
-Encryption is applied server-wide; it cannot be applied to selected users or 
-files.
+The main purpose of the Default Encryption Module is to protect users' files on 
+remote storage, and to do it easily and seamlessly from within ownCloud. 
 
-The Encryption app encrypts only the contents of files, and not filenames and 
-folder structures.
+The Default Encryption Module generates a strong encryption key, which is 
+unlocked by user's passwords. So your users don't need to track an extra 
+password, but simply log in as they normally do. It encrypts only the contents 
+of files, and not filenames and folder structures.
 
 You should regularly backup all encryption keys to prevent permanent data loss. 
 The encryption keys are stored in following folders:
```

Comment:
````data/owncloud_private_key```` and ````data/public-keys```` no longer exists.

We just have data/user/files_encryption for user specific keys and data/files_encryption for system wide keys